_ont/ont-zte-f6005.md

```diff
@@ -47,6 +58,115 @@ You can use this tool to generate the enable password:
 
 {% include cig_password.html username="ont" %}
 
+# Advanced settings
+## Unlock serial and TELNET
+
+This ONT is the twin brother of **CIG G97-CP**, if you can find its bootloader (named `CIG_bu.en_V3.09.15`), you can easly repack firmware and enable TTL. 
+You need a SPI programmer (that should be a 3.3V, like modded CH341a) to read and write back the flash.
+
+{% include alert.html content="This was tested on TIM V6.0.10N20 firmware!" alert="Note" icon="svg-info" color="blue" %}
+
+After you got a full dump of your ONT here is the procedure to cut&paste new bootloader and have TTL enabled:
+
+Cut old bootloader:
+```sh
+dd if=flash_dump.bin of=no_boot_flash_dump.bin bs=1 skip=184064
+```
+
+Attach new one:
+```sh
+cat CIG_bu.en_V3.09.15 no_boot_flash_dump.bin > mod-boot_flash_dump.bin
+```
+
+Now you can flash the file `mod-boot_flash_dump.bin` back to your SPI. 
+
+After power-up the ONT, the TTL will print this message:
+
+```sh
+**************************************
+*                                    *
+*  KEY -- Enter console terminal     *
+*                                    *
+**************************************
+```
+
+To access full U-Boot a special escape sequence is needed. 
+
+If you use TeraTerm software, create a simple MACRO file that contains this code:
+
+`send $1B $1D $0F $0B`
```

Comment:
Didn't you talk about it changing by version?

Reply:
what do you mean?